### PR TITLE
Added a load of app.imports

### PR DESCRIPTION
**Pull request by patocallaghan**
Increased the number of 3rd party dependencies for the app which would be more in line with a large-ish app

Also added autoprefixer support to app

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -12,8 +12,31 @@
     "jquery": "^2.1.4",
     "loader.js": "ember-cli/loader.js#3.2.0",
     "qunit": "~1.18.0",
-    "material-design-lite-src": "unicornext/material-design-lite#master",
+    "material-design-lite-src": "google/material-design-lite#master",
```
Comment (patocallaghan) [inline bower.json]: The `unicornext` repo no longer seems to exist

File: ember-cli-build.js [hunk 2]
```diff
@@ -24,5 +30,52 @@ module.exports = function(defaults) {
     destDir: 'assets/fonts'
   });
 
+  app.import('bower_components/ic-ajax/main.js', {
```
Comment (stefanpenner) [inline ember-cli-build.js]: This one is already present via an addon

Comment (patocallaghan) [inline ember-cli-build.js]: I've removed the `ic-ajax` import

**Comment by stefanpenner** [comment]
thanks

**Comment by stefanpenner** [comment]
updated readme: 

03dc7315cbad67bd39649d1df330e8870a91db13
bb35194cee0edf9a4a97e1a55553ff01806fcf0a

**Pull request merged**